src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -253,7 +253,7 @@
         <PackageId>$(Id)</PackageId>
         <PackageVersion>$(PackageVersion)</PackageVersion>
         <!-- Some packages support legacy portable profiles where dependencies are provided by targeting pack -->
-        <HarvestDependencies Condition="!$([System.String]::new('%(_FilesToPackage.TargetFramework)').StartsWith('portable-'))">true</HarvestDependencies>
+        <HarvestDependencies Condition="!$([System.String]::Copy('%(_FilesToPackage.TargetFramework)').StartsWith('portable-'))">true</HarvestDependencies>
```

Comment:
If you read the commit description you'll see:
>These calls to System.String.ctor were resulting in hundreds of first
chance exceptions in msbuild as it tried to find the correct overload.

>String.Copy doesn't have this problem since there are no overloads
to evaluate.